server/inMemoryStore.ts

```diff
@@ -0,0 +1,8 @@
+interface InMemoryStore {
+  userVersion: string
+}
+
+const inMemoryStore: InMemoryStore = {
```

Comment:
I think it would be helpful to document what sort of data we expect to keep in the `inMemoryStore`, how that differs from the data kept in `res.locals` and `req.session`

Reply:
Also discussed is that this approach does not work, as the in-memory storage is shared between all sessions. Will be updated.